trace:  Span in noop.Start is no longer allocated (#5457)
open-telemetry/opentelemetry-go: CHANGELOG.md

- Identify the `Meter` returned from the global `MeterProvider` in `go.opentelemetry.io/otel/global` with its schema URL. (#5426)
- Log a warning to the OpenTelemetry internal logger when a `Span` in `go.opentelemetry.io/otel/sdk/trace` drops an attribute, event, or link due to a limit being reached. (#5434)
- Document instrument name requirements in `go.opentelemetry.io/otel/metric`. (#5435)

> - `Start` in `go.opentelemetry.io/otel/trace/noop` span no longer heap allocated when the parent span is zero SpanContext. (#5457)

Reviewer: ```suggestion
- `Tracer.Start` in `go.opentelemetry.io/otel/trace/noop` no longer allocates a span for empty span context. (#5457)
```

Reviewer 2: I suggest also moving it to ` Changed` section (instead of `Fixed`).
